src/modules/Preferences/OrgPreferences/valueContainer/constants.ts

```diff
@@ -1,8 +1,13 @@
 import i18next from 'i18next';
```

Comment:
do you have any reason you didn't want to put it in the constants files?
FYI: i want to be synced with you.

Reply:
Yes, Replacing text with 'translation variable' for internationalization purpose was one of the DOD items. So I did it and in the future adding new languages for this page will be easy.